python/setup.py

```diff
@@ -192,6 +192,18 @@ def get_option_from_sys_argv(option_str):
 
     extra_compile_args = []
 
+    message_extra_link_args = None
```

Comment:
https://github.com/grpc/grpc/pull/24992/files
Could you add a comment like the one in grpc's PR to explain the reason?